cmd/collector/main.go

```diff
@@ -41,28 +41,33 @@ func main() {
 		return
 	}
 
-	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGTERM)
-	defer cancel()
+	logOpts, err := logOptions(loggingPath)
+	if err != nil {
+		log.Fatalf("Failed to get log options: %v", err)
+	}
 
-	settings, err := collector.NewSettings(*configPaths, version.Version(), nil)
+	// logOpts will override options here
+	logger, err := zap.NewProduction(logOpts...)
 	if err != nil {
-		log.Fatal(err)
+		log.Fatalf("Failed to set up logger: %v", err)
 	}
 
-	if err := run(ctx, *settings); err != nil {
-		log.Fatal(err)
+	col := collector.New((*configPaths)[0], version.Version(), logOpts)
```

Comment:
The collector use to be able to take in multiple config paths through creating a settings. Looks like we've lost that functionality here?

Reply:
We also had to set the set the `DisableGracefulShutdown` on the collector or else it would take over signal intercept from our process.

Reply:
`DisableGracefulShutdown` is set (col.Run calls collector.NewSettings, which sets this option)

As far as multiple configs, good point, I think I had it in my head that we were only using the first config previously, but we were using multiple. I'll change collector.New to accept the full slice.